packages/loot-core/src/server/budget/goaltemplates.js

```diff
@@ -329,19 +329,25 @@ async function applyCategoryTemplate(category, template_lines, month, force) {
         break;
       }
       case 'percentage': {
-        /*
-          let income_category = (await actual.getCategories()).filter(c => c.is_income == true && c.name == template.category);
-          let func = (getBudgetMonthTestFunc || getBudgetMonth);
-          let budget = await func(month);
-          for (var g = 0; g < budget.categoryGroups.length; g++) {
-            if (income_category.group_id == budget.categoryGroups[g].id) {
-              for (var c = 0; c < budget.categoryGroups[g].categories.length; c++)
-                if (income_category.id == budget.categoryGroups[g].categories[c].id) {
-                  let month_category = budget.categoryGroups[g].categories[c];
-                }
-            }
+        let percent = template.percent;
+        let monthlyIncome = 0;
+        if (template.category === 'all income') {
```

Comment:
the language as a whole is case insensitive, no reason to be different here (right?)

```suggestion
        if (template.category.toLowerCase() === 'all income') {
```